src/Components/ProductsPage.js

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import data from '../data.json';
+import ProductTable from './ProductTable';
+import SearchBar from './SearchBar';
+
+
+function ProductsPage () {
+const [products, setProducts] = useState(jsonData);
+const [productTable, setProductsTable] = useState(jsonData)
+const [SearchBar, setSearchBar] = useState()
+const [stock , SetStock]= useState(false)
+
+
+  return(
+      <div>
+        <h1>IronStore</h1>
+      </div>    
+      <div className="product-container">
```

Comment:
JSX must have only one top element. Here you have two (the div containing the h1 and the div `product-container`

Reply:
also, I don't see a closing tag for this div. All tags in JSX (except img, input and some others) need a closing tag